components/button/index.css

```diff
@@ -294,6 +294,17 @@ a.spectrum-Button {
 	}
 }
 
+/* Disable button label wrap  */
+.spectrum-Button--noWrap {
+	justify-content: flex-start;
+	
+	.spectrum-Button-label {
+		white-space: nowrap;
+		overflow: hidden;
+		text-overflow: ellipsis;
```

Comment:
Now that this is working like the text overflow behavior of Action button, it creates accessibility issues when there is text overflow, since the text can be sometimes unreadable. Does that mean we need to provide similar guidelines about a tooltip? We'd be asking more of the implementation for this outside the spec feature: https://spectrum.adobe.com/page/action-button/#Text-overflow

Reply:
@jawinn I tried it and it added the test case. 

Tooltip only shows when `nowrap` is true and `hideLabel` is false. 
If the text is not "ellipsized" or visible I added a class to hide the tooltip: `spectrum-Tooltip--hide`.